pkg/virtctl/guestfs/guestfs.go

```diff
@@ -614,22 +605,26 @@ func createAttacher(client *K8sClient, p *corev1.Pod, command string) error {
 		"If you don't see a command prompt, try pressing enter.", resChan)
 }
 
-func (client *K8sClient) createInteractivePodWithPVC(pvc, image, ns, command string, args []string, isblock bool) error {
-	pod, err := createLibguestfsPod(pvc, image, command, args, kvm, isblock)
+func (c *guestfsCommand) createInteractivePodWithPVC(client *K8sClient, ns, command string, args []string, isblock bool) error {
```

Comment:
Maybe in a follow up: This method's signature could be simplified a lot since it is used only in a single place and it gets passed a constant and an empty slice.